specification/cognitiveservices/OpenAI.Inference/models/completions/streaming/chat_completions.tsp

```diff
@@ -0,0 +1,126 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@typespec/versioning";
+
+import "../azure_chat_extensions.tsp";
+import "../common.tsp";
+import "../functions.tsp";
+import "../chat_completions.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+
+namespace Azure.OpenAI;
+
+@doc("A single, role-attributed message within a chat completion interaction.")
+model ChatMessageStreamChunk {
+  @doc("The role associated with this message payload.")
+  @projectedName("json", "role")
+  role: ChatRole;
+
+  #suppress "@azure-tools/typespec-azure-core/no-nullable" "we explicitly want a nullable string"
+  @doc("The text associated with this message payload.")
+  @projectedName("json", "content")
+  content?: string; // they key might not be present on a streaming scenario
```

Comment:
Can remove the comments for readability in posterity, particularly in light of the `#supress` justification